src/main/java/com/divudi/bean/common/BillController.java

```diff
@@ -3167,6 +3168,32 @@ public void updateReferingDoctor() {
         billFacade.edit(bill);
         JsfUtil.addSuccessMessage("Ref Doctor Updated");
     }
+    
+    public String findBatchBillSessionID(String deptID){
```

Comment:
We can get the id to search here instead of deptID
Also I think we can get it from the object without using JPQL

Please discuss